src/sasjs-job/execute.js

```diff
@@ -0,0 +1,158 @@
+import chalk from 'chalk'
+import ora from 'ora'
+import { displayResult } from '../utils/displayResult'
+import { createFile, createFolder, folderExists } from '../utils/file-utils'
+import path from 'path'
+
+/**
+ * Triggers existing job for execution.
+ * @param {object} sasjs - configuration object of SAS adapter.
+ * @param {string} accessToken - an access token for an authorized user.
+ * @param {string} jobPath - location of the job on SAS Drive.
+ * @param {object} target - SAS server configuration.
+ * @param {boolean} waitForJob - flag indicating if CLI should wait for job completion.
+ * @param {boolean} output - flag indicating if CLI should print out job output. If string was provided, it will be treated as file path to store output.
+ * @param {boolean | string} output - flag indicating if CLI should print out job output. If string was provided, it will be treated as file path to store output. If filepath wasn't provided, output.json file will be created in current folder.
+ * @param {boolean | string} log - flag indicating if CLI should fetch and save log to the local folder. If filepath wasn't provided, {job}-log.json file will be created in current folder.
```

Comment:
aha - this description is clearer.  It's important to distinguish between a filepath (which points to a file) and a folder path.  If the `log` flag is provided with a value, most times I expect it would be to a filepath (eg `./folder/file.log`).  However if we can easily support a folder (eg `./folder`) as well, and it will apply this naming convention, that's great for ease of use. 

Primarily though we should support / document the full filepath (folder + filename) approach, it's the most intuitive / in line with behaviour in the SAS world

Reply:
Currently:
1. if folder path was provided for `-o ./someFolder` or `-l ./someFolder`, cli will create files `./someFolder/[output|log].json` 
2. if file path was provided for `-o ./someFolder/customName.json` or `-l ./someFolder/customName2.json`, cli will create files`./someFolder/customName[2].json`